### PR TITLE
New "What is kubernetes" topic

### PR DESCRIPTION
**Pull request by RichieEscarez**
@brendandburns @thockin @bgrant0607 

This is mostly copied from other overviews we have with slight edits (please ensure they are accurate)

### CODE REVIEW

File: docs/whatisk8s.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
+# What is Kubernetes?
+
+Kubernetes is an open source container cluster orchestration platform that was started by Google in 2014.
+
+Use Kubernetes to manage your containerized applications across multiple hosts through automated deployment, container life-cycle maintenance, and application scaling.
+
+With Kuberenets, you are able to quickly and efficiently respond to customer demand:
```
Comment (thockin) [inline docs/whatisk8s.md]: misspelled kubernetes

**Comment by thockin** [comment]
I'm not the wordsmith that Brian is.

@bgrant0607

**Comment by bgrant0607** [comment]
Looking

File: docs/whatisk8s.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Kubernetes is an open source container cluster orchestration platform that was started by Google in 2014.
+
+Use Kubernetes to manage your containerized applications across multiple hosts through automated deployment, container life-cycle maintenance, and application scaling.
+
+With Kuberenets, you are able to quickly and efficiently respond to customer demand: 
+
+ - Scale your applications on the fly
+ - Seamlessly roll out new features
+ - Maintain optimal hardware use (only use the resouces you need)
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: Optimize use of your hardware by using only the resources you need

(grammar + typo)

File: docs/whatisk8s.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+
+# What is Kubernetes?
+
+Kubernetes is an open source container cluster orchestration platform that was started by Google in 2014.
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: 1. I think the term "orchestration" does us a disservice. We should avoid it.
2. "open source container cluster orchestration" is an unhyphenated run-on adjective.
3. The project was started in 2014, not the platform.

The Kubernetes project was started by Google in 2014. Kubernetes is an open-source platform for automating deployment, scaling, and operations of application containers. It schedules containers across clusters of hosts.

File: docs/whatisk8s.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+ - Scale your applications on the fly
+ - Seamlessly roll out new features
+ - Maintain optimal hardware use (only use the resouces you need)
+
+
+#### Kubernetes is:
+
+* **lean**: lightweight, simple, accessible
+* **portable**: public, private, hybrid, multi cloud
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: multi-cloud

**Comment by bgrant0607** [comment]
This is a transliteration of my wiki page: https://github.com/GoogleCloudPlatform/kubernetes/wiki/Why-Kubernetes%3F

Once this is merged, we could probably just link from the wiki to this page on kubernetes.io.

**Comment by bgrant0607** [comment]
Problems have been fixed in gh-pages: #11644

**Comment by RichieEscarez** [comment]
@bgrant0607 ive integrated your edits (thank you!) and made several changes to the 2nd half, PTAL

File: docs/whatisk8s.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# What is Kubernetes?
+
+The Kubernetes project was started by Google in 2014.
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: I moved this down in a subsequent PR. It logically belongs with the builds-on-our-experience sentence. And it's really awkward to have it first.

File: docs/whatisk8s.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+# What is Kubernetes?
+
+The Kubernetes project was started by Google in 2014. 
+
+Kubernetes is an open-source platform for automating deployment, scaling, and operations of application containers across clusters of hosts. Kubernetes builds upon a [decade and a half of experience that Google has with running production workloads at scale](https://research.google.com/pubs/pub43438.html), combined with best-of-breed ideas and practices from the community.
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: I don't think condensing this text into a single paragraph is an improvement. It's harder to parse.

File: docs/whatisk8s.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+* **Environmental consistency across development, testing, and production**:  
+    Runs the same on a laptop as it does in the cloud.
+* **Cloud and OS distribution portability**:  
+    Runs on Ubuntu, RHEL, on-prem, or Google Container Engine, which makes sense for all environments: build, test, and production.
+* **Resource isolation**:  
+    Predictable application performance.
+* **Resource utilization**:  
+    High efficiency and density.
+
+[Article about why Spotify chose containers.](http://www.pwc.com/us/en/technology-forecast/2014/cloud-computing/features/open-source-application-deployment-containers.jhtml)
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: We could probably remove this from here and put it wherever else we have customer/partner stories/quotes.

File: docs/whatisk8s.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+[Background on what containers are.](http://aucouranton.com/2014/06/13/linux-containers-parallels-lxc-openvz-docker-and-more/)
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: We could probably just link to this from the first mention of "containers" in this subsection.

File: docs/whatisk8s.md (hunk 5 above)
Comment (RichieEscarez) [inline docs/whatisk8s.md]: done

File: docs/whatisk8s.md (hunk 6 above)
Comment (RichieEscarez) [inline docs/whatisk8s.md]: separated both sentences

File: docs/whatisk8s.md (hunk 7 above)
Comment (RichieEscarez) [inline docs/whatisk8s.md]: This link still exists in the wiki @ https://github.com/GoogleCloudPlatform/kubernetes/wiki/Why-Kubernetes%3F. Ill remove it from this new document and when the time comes, ensure it gets moved into a better location under k8s.io.

File: docs/whatisk8s.md (hunk 8 above)
Comment (RichieEscarez) [inline docs/whatisk8s.md]: Moved here:
Looking for reasons why you should be using [containers](http://aucouranton.com/2014/06/13/linux-containers-parallels-lxc-openvz-docker-and-more/)?

**Comment by RichieEscarez** [comment]
All changes made. @bgrant0607 PTAL

File: docs/whatisk8s.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+ - Optimize use of your hardware by using only the resources you need.
+
+#### Kubernetes is:
+
+* **lean**: lightweight, simple, accessible
+* **portable**: public, private, hybrid, multi-cloud
+* **extensible**: modular, pluggable, hookable, composable
+* **self-healing**: auto-placement, auto-restart, auto-replication
+
+The Kubernetes project was started by Google in 2014.
```
Comment (bgrant0607) [inline docs/whatisk8s.md]: Doesn't make sense here. Why not just copy my changes from gh-pages. I can do it if it would be easier

**Comment by RichieEscarez** [comment]
@bgrant0607, sorry, i misunderstood your previous comments. Paragraph now copied from gh-pages.

**Comment by bgrant0607** [comment]
Thanks.

Please run hack/run-gendocs.sh now.

**Comment by RichieEscarez** [comment]
@bgrant0607 Ran hack/run-gendocs.sh and swashed commit. It should be ready now.

**Comment by bgrant0607** [comment]
Obsolete.

**Pull request closed without merging**